### PR TITLE
Feature/4592 browser back workflow first screen

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-4592

#### Changes proposed in this Pull Request:

- Added navigate browser back on the first screen of registration workflow
- Refactored children of registration workflow logic
- Updated test cases

#### Screenshots / Screen Recording (if applicable)

https://github.com/etn-ccis/blui-react-workflows/assets/120575281/c34c480e-9210-49a7-9dbf-1bb556d7175b

#### To Test:
- cd login
- yarn start:example
- Click on Create Account button
- Click on Back button on the first screen
- It should navigate to previous page in the browser

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by manojleaton** [comment]
Working fine and code looks good.

File: login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { useEffect, useState } from 'react';
+import React, { cloneElement, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router';
```
Comment (joebochill) [inline login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx:2]: Our workflow package is router independent...we can't use react-router inside it.

Comment (surajeaton) [reply login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx]: removed

**Comment by surajeaton** [comment]
@manojleaton @joebochill Please review.

**Comment by manojleaton** [comment]
Working fine!

**Review by manojleaton — APPROVED** [review]

**Pull request merged**